scripts/global.js

```diff
@@ -1120,18 +1133,13 @@ async function renderProposals(arrProposals, fContested) {
                 if (
                     cProposal.IsEstablished &&
                     nNetYes >= nRequiredVotes &&
-                    totalAllocatedAmount + cProposal.MonthlyPayment <=
+                    totalAllocatedAmount + nMonthlyPayment <=
                         cChainParams.current.maxPayment / COIN
                 ) {
-                    strFundingStatus = translation.proposalFunded;
-                    totalAllocatedAmount += cProposal.MonthlyPayment;
+                    totalAllocatedAmount += nMonthlyPayment;
```

Comment:
An I missing something or this whole if is useless? In case remove it and move the line  
`totalAllocatedAmount += nMonthlyPayment;`
after 
`// Scenario 4: Enough votes, and established`

Reply:
Good catch, merged this with the prior "proposal state" logic and it's much cleaner.

Also removed the horrible slow "`domTable.id == 'proposalsTableBody'`" check, which was just a longer version of `!fContested`.